### ISSUE TITLE
Deprecate `runner` in favor of `taskRunner` and document usage

### ISSUE THREAD

**Issue by wrussell1999** [kestra-io/plugin-spark#72]
Does `taskRunner` work for this task? If so, we should replace `runner` with it.

Same for `docker` with `taskRunner` and `containerImage`

_Originally posted by @wrussell1999 in https://github.com/kestra-io/plugin-spark/pull/71#discussion_r1733019022_

**Comment by loicmathieu** [comment]
Yes, it is supported, it seems it is only a documentation issue.
